core/tests/protractor_utils/ModeratorPage.js

```diff
@@ -0,0 +1,37 @@
+// Copyright 2021 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Page object for the admin page, for use in Protractor
+ * tests.
+ */
+
+var general = require('./general.js');
+var waitFor = require('./waitFor.js');
+
+var ModeratorPage = function() {
+  var moderatorPage = element(by.css('.protractor-test-moderator-page'));
+
+  this.get = async function() {
+    await browser.get(general.MODERATOR_URL_SUFFIX);
```

Comment:
Why is the moderator page URL defined in general?

Reply:
`general.js` defines a bunch of the common URL suffixes:

https://github.com/oppia/oppia/blob/ff552709ee514030fcb41eb0fe7da5657eab1c61/core/tests/protractor_utils/general.js#L77-L84